src/viadot/orchestration/prefect/flows/vid_club_to_adls.py

```diff
@@ -0,0 +1,97 @@
+"""Download data from Vid CLub API and load it into Azure Data Lake Storage."""
+
+from typing import Any, Dict, List, Literal
+from prefect import flow
+from prefect.task_runners import ConcurrentTaskRunner
+from viadot.orchestration.prefect.tasks import df_to_adls, vid_club_to_df
+
+
+@flow(
+    name="Vid CLub extraction to ADLS",
+    description="Extract data from Vid CLub and load it into Azure Data Lake Storage.",
+    retries=1,
+    retry_delay_seconds=60,
+    task_runner=ConcurrentTaskRunner,
+)
+def vid_club_to_adls(
+    *args: List[Any],
+    source: Literal["jobs", "product", "company", "survey"] = None,
```

Comment:
There's a standard term for this so I would just use that

```suggestion
    endpoint: Literal["jobs", "product", "company", "survey"] = None,
```

Reply:
[39e2e98](https://github.com/dyvenia/viadot/pull/1044/commits/39e2e9848abbcf0071fb95cca100843d31554257)